FHEW.cpp

```diff
@@ -8,8 +8,11 @@ using namespace std;
 namespace FHEW {
 
   typedef Ring_ModQ ct_ModQ[K2][2];   // Ciphertext in coefficient form
+  typedef Ring_ModQ ct_ModQ1[2];   // Entry C[1] of ct_ModQ
   typedef Ring_ModQ dct_ModQ[K2][K2]; // Decomposed Ciphertext in coeff form
+  typedef Ring_ModQ dct_ModQ1[K2]; // Entry C[1] of dct_ModQ
   typedef Ring_FFT  dct_FFT[K2][K2];  // Decomposed Ciphertext in FFT form
+  typedef Ring_FFT  dct_FFT1[K2];  // Entry C[1] of dct_FFT
   
```

Comment:
I see no obvious reason to keep old types (they seems to only be used in this file). Did I miss something